src/data/faq.json

```diff
@@ -2171,66 +2171,304 @@
+            ],
+            "G": [
+                {
+                    "term": "GitHub",
+                    "anchor": "github",
+                    "description": "GitHub is a platform that allows many different people to work on a project at the same time. It can merge everyone's contributions and documents any changes."
+                }
+            ],
+            "I": [
+                {
+                    "term": "iOS",
+                    "anchor": "ios",
+                    "description": "iOS is a mobile operating system developed by Apple for the iPhone, iPod touch and formerly also iPad. Along with <a href='#glossary_android'>Android</a>, it is one of the two operating systems that the Corona-Warn-App supports."
+                }
+            ],
+            "M": [
+                {
+                    "term": "Molecular biological test",
+                    "anchor": "molecular_biological_test",
+                    "description": "See <a href='#glossary_pcr_test'>PCR test</a>"
+                }
+            ],
+            "O": [
+                {
+                    "term": "Open-Source",
+                    "anchor": "open_source",
+                    "description": "Code that is visible, accessible, modifiable and distributable to the public."
                 }
             ],
             "P": [
                 {
-                    "term": "Proof of vaccination",
-                    "anchor": "vaccination_proof",
-                    "description": "The proof of vaccination documents a vaccination procedure with a vaccine recommended by the Paul Ehrlich Institute. A vaccination proof must contain at least the following attributes: <ul><li>the personal data of the vaccinated person (at least surname, first name and date of birth)</li><li>date of vaccination, number of vaccinations,</li><li>designation of the vaccine,</li><li>name of the disease against which the vaccination was carried out, and</li><li>characteristics that indicate the person or institution responsible for carrying out the vaccination or issuing the proof of vaccination, for example, an official symbol or the name of the issuer. </li></ul> <br/><br/> For entry, according to § 2 number 10 Coronavirus-Einreiseverordnung, the vaccination certificate must still meet additional requirements (regarding languages and vaccination schema). <br/><br/> The proof of vaccination can be documented in different ways, one form is the digital <a href='#glossary_vaccination_certificate'>vaccination certificate</a>."
+                    "term": "PCR Test",
+                    "anchor": "pcr_test",
+                    "description": "PCR tests are considered the most accurate among corona tests. In this type of test, medical personnel collect the sample and have it analyzed by a laboratory."
+                },
+                {
+                    "term": "Proof of Vaccination",
+                    "anchor": "proof_of_vaccination",

```

Comment:
Changing the anchor from "vaccination_proof" to "proof_of_vaccination" in a section which has already been published breaks any bookmarks stored by users.

If somebody bookmarked https://www.coronawarn.app/en/faq/#glossary_vaccination_proof that will not work any more.

Reply:
Will be changed in the next commit, thanks! I initially submitted it with the "vaccination_proof" anchor, but it somehow ended up being changed by others. I now made sure that it wouldn't cause any issues to revert it to "vaccination_proof" and changed any hyperlinks linking to that glossary entry.